docs/Splashkit/Applications/Arcade Machines/Arcade Machine Setup/Setup Arcade Machine.md

```diff
@@ -126,20 +125,28 @@ and one to wpa_supplicant:
      curl -sSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin --channel STS
      ```
 
-  2. Add the following to the end of the file `~/.bashrc` with these commands
+  1. Add the following to the end of the file `~/.bashrc` with these commands
 
      ```shell
      echo 'export DOTNET_ROOT=$HOME/.dotnet' >> ~/.bashrc
      echo 'export PATH=$PATH:$HOME/.dotnet' >> ~/.bashrc
+     echo 'export DOTNET_ROOT=$HOME/.dotnet' >> ~/.bash.bash_profile
+     echo 'export PATH=$PATH:$HOME/.dotnet' >> ~/.bash_profile
```

Comment:
We _could_ source .bashrc from .bash_profile instead of duplicating export commands. If we do this, we must ensure that we don't make an infinite loop (.bash_profile sources .bashrc which sources .bash_profile...)

Not sure what's already in .bashrc - what do you think?

Reply:
Would that cause weird behavior if the user configured their Arcade Machine with a desktop environment?  or are you suggesting that we setup bashrc to clone its config directly from .bash_profile? 

(I'm not exceptionally familiar with  .bash_profile as I typically setup everything under my local .bashrc)

Reply:
I believe that you just put in `source ~/.bashrc` and it gets executed when .bash_profile loads so you can kind of chain them.

My understanding is that .bash_profile loads when you use a non-root shell.

Reply:
Yeah it's a bit of a mess because history. Here's an excerpt from the bash man page

```
       ~/.bash_profile
              The personal initialization file, executed for login shells
       ~/.bashrc
              The individual per-interactive-shell startup file
```

So bash_profile is executed once when the user logs in (the definition of log in here is pretty vague so it ends up getting ran pretty frequently). bashrc is executed every time a new interactive session starts.

My recommendation would be to put the environment variable setting in bashrc. It shouldn't cause any weirdness with the desktop environment.